apcd-cms/src/apps/view_users/templates/view_user_modal.html

```diff
@@ -1,30 +1,39 @@
-<div id="viewRegistrationModal_{{r.reg_id}}" class="modal fade" role="dialog">
+<div id="viewUserModal_{{r.user_number}}" class="modal fade" role="dialog">
   <div class="modal-dialog modal-lg">
 
     <!-- Modal content-->
     <div class="modal-content">
       <div class="modal-header">
-        <h4 class="modal-title">User Notes</h4>
+        <h4 class="modal-title">User Info</h4>
         <button type="button" class="close" data-dismiss="modal">
           <span aria-hidden="true">&#xe912;</span>
         </button>
       </div>
       <div class="modal-body">
         <!-- <p>{{modal_content}}</p> -->
         <div>
-          <dl>
-            <h4>Organization</h4>
-            {{r.view_modal_content.city}}
-            <hr>
-            <h4>Roles</h4>
-            
-            <hr>
-            <h4>Notes</h4>
-                        
-            <hr>
-            <h4>Contact Information</h4>
-            
-          </dl>
+          <dl class="c-data-list--is-vert c-data-list--is-wide">
+                <dt class="c-data-list_key">User ID</dt>
+                <dd class="c-data-list__value">{{r.user_id}}</dd>
+                <dt class="c-data-list_key">User Number</dt>
+                <dd class="c-data-list__value">{{r.user_number}}</dd>
+                <dt class="c-data-list_key">Email</dt>
```

Comment:
```suggestion
                <dt class="c-data-list__key">User ID</dt>
                <dd class="c-data-list__value">{{r.user_id}}</dd>
                <dt class="c-data-list__key">User Number</dt>
                <dd class="c-data-list__value">{{r.user_number}}</dd>
                <dt class="c-data-list__key">Email</dt>
```
These field labels in modal are missing trailing colons, just a class name issue.